src/mods/search/toggle.rs

```diff
@@ -0,0 +1,95 @@
+use crate::{
+    linquebot::{msg_context::Context, types::Consumption, Module, ModuleDescription, ModuleKind},
+    utils::telegram::prelude::WarnOnError,
+};
+use serde::{Deserialize, Serialize};
+use teloxide_core::{prelude::Request, types::Message};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Search {
+    pub search_enabled: bool,
+    pub search_recording_enabled: bool,
+}
+
+impl Search {
+    pub fn default() -> Self {
+        Search {
+            search_enabled: false,
+            search_recording_enabled: false,
+        }
+    }
+}
```

Comment:
It may be better to impl the `Default` trait, or, if treating the config as defaultable object is undesired, just use the name `new`.